Thyra Collection API Testing.postman_collection.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
If you want your postman collection to be executed by the CI, you should move this file in `api/test`

Reply:
To be addressed in #504